lib/ansible/galaxy/data/default/collection/README.md.j2

```diff
@@ -1,3 +1,4 @@
+#SPDX-License-Identifier: MIT-0
```

Comment:
I think we should create `README.md.license` files instead of trying to inject into the file, since this is basically documentation, it would likely confuse users of the collection, thinking it is the license of the collection:

https://reuse.software/spec/#comment-headers

Reply:
that is the COPYING file i created, should we just leave that and remove all the header changes?

Reply:
The `COPYING` file is in our repo, and won't get copied into the `init`ed directory.  I think a `.license` file will be the best way to handle it.  And we wouldn't want to put the `COPYING` file into the users collections, to avoid confusion about the license.

Reply:
no i'm confused, you want to add license to the product of init, but not really to override the user's license?

Reply:
Correct, the licenses for the files should be included in the product of init, so in the case of markdown, where there is no standard for comments, we should instead of putting it in the `README.md.j2` we should create a new `README.md.license` with the license, per the link I put up a few comments.  Sorry if I'm not being clear.

Reply:
Need to remove this license I believe.